recipes/muparser/build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+sh ./configure --prefix=$PREFIX --disable-samples
+make -j2
```

Comment:
@183amir It's generally bad to try $CPU_COUNT - because that is not reliably set to the CPU limit that the CI's apply.  @jschueller's limit of 2 is good here.  I would only suggest adding a link to any Travis and/or CircleCI docs that specify that 2 cpus are available.